client/blocks/reader-full-post/index.jsx

```diff
@@ -235,7 +235,14 @@ export class FullPostView extends React.Component {
 	attemptToSendPageView = () => {
 		const { post, site } = this.props;
 
-		if ( post && post._state !== 'pending' && site && site.ID && ! this.hasSentPageView ) {
```

Comment:
this is one ugly check. 
I could see us adding some common utils later for: `isValidPost` and `isValidSite`